datalad_container/containers_run.py

```diff
@@ -0,0 +1,112 @@
+
+__docformat__ = 'restructuredtext'
+
+import logging
+from simplejson import loads
+from argparse import REMAINDER
+
+from datalad.interface.base import Interface
+from datalad.interface.base import build_doc
+from datalad.support.param import Parameter
+from datalad.distribution.dataset import datasetmethod
+from datalad.distribution.dataset import require_dataset
+from datalad.interface.utils import eval_results
+
+from datalad.interface.run import Run
+from datalad_container.containers_list import ContainersList
+
+lgr = logging.getLogger("datalad.containers.containers_run")
+
+
+_run_params = dict(
+    Run._params_,
+    container_name=Parameter(
+        args=('-n', '--container-name',),
+        metavar="NAME",
+        doc="""Specify the name of a known container to use for execution,
+        in case multiple containers are configured."""),
+    # have to override the cmd parameter, because in `run` it says
+    # SHELL COMMAND but with a container it could be something totally
```

Comment:
I don't know that the SHELL in run's `metavar='SHELL COMMAND'` really adds that much.  Perhaps we should just change it to `metavar='COMMAND'` there and make sure Run's class docstring mentions explicitly that it's a shell command.

Reply:
That would be a solution.